src/genpy/message.py

```diff
@@ -228,7 +233,7 @@ def check_type(field_name, field_type, field_val):
             elif not type(field_val) == bytes:
                 raise SerializationError('field %s must be of type bytes or an ascii string'%field_name)
         else:		
-            if type(field_val) == unicode:
+            if type(field_val) == unicode:  # noqa
```

Comment:
Please only ignore the specific code reported on this line.

Reply:
It accelerates our ability to find and fix errors if linters do not raise unnecessary errors.  An alternative would be to use Python porting best practice __(use feature detection instead of version detection)[https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection) but this codebase does not follow that advise.

Reply:
> It accelerates our ability to find and fix errors if linters do not raise unnecessary errors.

Please use `# noqa: F821` to only ignore the specific error message but not hide potentially other errors in this line.